### ISSUE TITLE
Intermittent timeout failure in e2e right-click-menus.spec.ts

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#3128]
## tl;dr

The changes in #3127 seem to have introduced an intermittent failure in e2e test suite `right-click-menus.spec.ts`.

## Details

There was [a recent e2e test failure in CI](https://github.com/brimdata/zui/actions/runs/9895704643). I managed to repro the failure locally and have confirmed it started happening in commit a32bb54 which is associated with the changes in #3127. At that commit I did this looping run of the tests in question:

```
$ SUCCESSES=0; FAILURES=0; while true; do   yarn e2e -g right-click-menus.spec.ts --skip-nx-cache; if [ $? -eq 0 ] ; then SUCCESSES=$(expr $SUCCESSES + 1); else FAILURES=$(expr $FAILURES + 1); fi; echo "Successful test runs=$SUCCESSES, failed test runs=$FAILURES"; done
```

That resulted in 11 successful runs and 7 failed runs before I stopped the loop. By comparison, at the previous commit 8ba4a4b the same loop made it through 20 successful runs and 0 failures before I stopped it.

An example of how it looks when it fails:

```
 >  NX   Ran target test for project zui-player (47s)
 
         With additional flags:
           --g=right-click-menus.spec.ts
 
    ✖    1/1 failed
    ✔    0/1 succeeded [0 read from cache]
 
Successful test runs=1, failed test runs=6

> nx run zui-player:test -g right-click-menus.spec.ts

Running 9 tests using 1 worker
  ✓  1 right-click-menus.spec.ts:12:3 › right-click-menus › Inspector node menu (645ms)
  ✓  2 right-click-menus.spec.ts:23:3 › right-click-menus › table cell menu (883ms)
  ✓  3 right-click-menus.spec.ts:31:3 › right-click-menus › sort asc (620ms)
  ✓  4 right-click-menus.spec.ts:39:3 › right-click-menus › sort desc (891ms)
  ✓  5 right-click-menus.spec.ts:48:3 › right-click-menus › hide table column (688ms)
  ✓  6 right-click-menus.spec.ts:56:3 › right-click-menus › who is (2.3s)
  ✓  7 right-click-menus.spec.ts:66:3 › right-click-menus › virus total (929ms)
  ✓  8 right-click-menus.spec.ts:74:3 › right-click-menus › inspector show details (999ms)
  ✘  9 right-click-menus.spec.ts:84:3 › right-click-menus › table show details (30.0s)
  1) right-click-menus.spec.ts:84:3 › right-click-menus › table show details ───────────────────────
    Test timeout of 30000ms exceeded.
    Error: locator.click: Application exited
    Call log:
      - waiting for getByRole('gridcell', { name: 'conn', exact: true }).first()
      -   locator resolved to <span role="gridcell" class="zeek-path-tag conn-bg-col…>conn</span>
      - attempting click action
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #1
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #2
      -   waiting 20ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #3
      -   waiting 100ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #4
      -   waiting 100ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #5
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #6
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #7
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #8
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #9
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #10
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #11
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #12
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #13
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #14
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #15
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #16
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #17
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #18
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #19
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #20
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #21
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #22
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #23
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #24
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #25
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #26
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #27
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #28
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #29
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #30
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #31
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #32
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #33
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #34
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #35
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #36
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #37
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #38
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #39
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #40
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #41
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #42
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #43
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #44
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #45
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #46
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #47
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #48
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #49
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #50
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #51
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #52
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #53
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #54
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #55
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #56
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <html>…</html> intercepts pointer events
      - retrying click action, attempt #57
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
      -   element is visible, enabled and stable
      -   scrolling into view if needed
      -   done scrolling
      -   <canvas width="14" height="90" aria-hidden="true" class=…></canvas> from <main id="main-area" role="tabpanel" class="sc-cTQhss …>…</main> subtree intercepts pointer events
      - retrying click action, attempt #58
      -   waiting 500ms
      -   waiting for element to be visible, enabled and stable
       at ../helpers/test-app.ts:197
      195 |
      196 |   async rightClick(role: Role | RegExp, name?: string, options?: object) {
    > 197 |     return this.locate(role, name, options).click({ button: 'right' });
          |                                             ^
      198 |   }
      199 |
      200 |   async attached(role: Role | RegExp, name?: string, options?: object) {
        at TestApp.rightClick (/Users/phil/work/zui/packages/zui-player/helpers/test-app.ts:197:45)
        at /Users/phil/work/zui/packages/zui-player/tests/right-click-menus.spec.ts:87:15
  Slow test file: right-click-menus.spec.ts (37.9s)
  Consider splitting slow test files to speed up parallel execution
  1 failed
    right-click-menus.spec.ts:84:3 › right-click-menus › table show details ────────────────────────
  8 passed (43.7s)

 —————————————————————————————————————————————————————————————————————————————

 >  NX   Ran target test for project zui-player (46s)
 
         With additional flags:
           --g=right-click-menus.spec.ts
 
    ✖    1/1 failed
    ✔    0/1 succeeded [0 read from cache]
```
